.github/workflows/preview-release-on-comment.yml

```diff
@@ -79,11 +81,12 @@ jobs:
             --body "Release workflow ${{ job.status == 'success' && 'succeeded ✅' || 'failed ❌' }}\nSee details: [Workflow Run](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-            
+
       - name: Set latest commit status as ${{ job.status }}
-        uses: myrotvorets/set-commit-status-action@master
+        uses: myrotvorets/set-commit-status-action@v2
         if: always()
         with:
           sha: ${{ steps.comment-branch.outputs.head_sha }}
           token: ${{ steps.generate_github_token.outputs.token }}
           status: ${{ job.status }}
+          allowForks: false
```

Comment:
In case other reviewers don't know about this property (as me), here is its description:
> Whether to allow to set commit status if the request comes from a forked repository. Defaults to false because the GITHUB_TOKEN for a forked repository usually does not have the necessary permissions to update the commit status. This only affects pull requests.

Reply:
nice, thanks